bugfix: Conveyor Belt and Disposal Outlet on Cyberiad & Celestation.

PiroMage opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание

<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->
Изменение айди у конвейеров карго шаттла, чтобы были соединены с рычагом на кибериаде.
Изменение конвейеров на карте Селестии, а также изменены выбрасыватели у мусорки, чтобы не выкидывало вещи в стекло и не ломало его.

## Ссылка на баг-репорт
https://discord.com/channels/617003227182792704/943940908162875473/1301564698327781396
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Тесты
Не требуется.
<!-- Как вы тестировали свой код. Ревьюеру будет проще, если он будет знать какие действия вы совершали, проверяя свой ПР. -->

PiroMage committed:
Починка лент и выплевывателей

1) Законекчены ленты к рычагам в карго на карте Кибериада.
2) Изменены и починены ленты с выплевывателями на карте Целестатион.
3) Изменен выплевыватель на Нове.

Daeberdir commented on _maps/map_files/celestation/celestation.dmm:
```diff
@@ -101469,6 +101471,13 @@
 	icon_state = "whitepurple"
 	},
 /area/toxins/misc_lab)
+"sio" = (
+/obj/structure/disposaloutlet{
+	dir = 8;
+	eject_range = 2
+	},
+/turf/space/openspace,
+/area/space)
```
![image](https://github.com/user-attachments/assets/f48ac2fc-9a88-4be9-86a3-f6a4b644ccc9)

PiroMage committed:
Откат изменений Новы

Перенос изменений патча Новы из общего багфикса конвейеров в патч Новы.

PiroMage committed:
Фикс карты

1) Убрана мусорка на месте прилета шаттла карго.
2) Изменены конвейерные ленты в мусорке карго.
